src/mlo_dir_conv.cpp

```diff
@@ -162,7 +162,8 @@ static auto GetImplicitGemmWrWSolvers()
                                            miopen::solver::ConvHipImplicitGemmV4R4GenWrWXdlops,
                                            miopen::solver::ConvHipImplicitGemmV4R1WrW,
                                            miopen::solver::ConvHipImplicitGemmV4R4WrW,
-                                           miopen::solver::ConvAsmImplicitGemmV4R1DynamicWrw>{};
+                                           miopen::solver::ConvAsmImplicitGemmV4R1DynamicWrw,
+                                           miopen::solver::ConvHipImplicitGemmWrwV4R4Xdlops>{};
```

Comment:
Order is important. This will never be used, most likely. See `WORKAROUND_SWDEV_227826`.

Reply:
@ltqin Please move your solver to be the first bwd-wrw solver in this list, otherwise it will not be picked by default

Reply:
I am assuming it is expected to be the fastest (on average) when applicable, right?

Reply:
@atamazov thanks,fixed it